src/main/java/uk/gov/companieshouse/alphabeticalcompanysearchconsumer/service/AlphabeticalIndexUpdaterService.java

```diff
@@ -31,6 +34,19 @@ public void processMessage(ServiceParameters parameters) {
             ", resource kind " + resourceKind + ", resource URI " + resourceUri + ".",
             getLogMap(message));
 
+            try {
+                upsertService.upsertService(parameters);
```

Comment:
Could we make this more specific? 
Add if statements to ensure that the event.type is correct. 
Take a look in advanced search equivalent for example.

Reply:
will take a look at this now